src/import/v8cpuFormatter.ts

```diff
@@ -0,0 +1,61 @@
+import {CPUProfile, CPUProfileNode} from './chrome'
+
+export interface OldCPUProfileNode {
+  functionName: string
+  lineNumber: number
+  scriptId: string
+  url: string
+  hitCount: number
+  bailoutReason: string
+  id: number
+  children: OldCPUProfileNode[]
+}
+
+export interface OldCPUProfile {
+  startTime: number
+  endTime: number
+  head: OldCPUProfileNode
+  samples: number[]
+  timestamps: number[]
+}
+
+function treeToArray(node: OldCPUProfileNode, nodes: CPUProfileNode[]) {
```

Comment:
This is a very minor thing, but I find functions that both mutate their arguments *and* return a new value confusing. I think this could be avoided like so:

```
function treeToArray(root: OldCPUProfileNode): CPUProfileNode[] {
  const nodes: CPUProfileNode[] = []
  function visit(node: CPUProfileNode) {
    nodes.push({
      id: node.id,
      callFrame: {
        columnNumber: 0,
        functionName: node.functionName,
        lineNumber: node.lineNumber,
        scriptId: node.scriptId,
        url: node.url,
      },
      hitCount: node.hitCount,
      children: node.children.map(child => child.id),
    })
    nodes.children.forEach(visit)
  }
  visit(root)
  return nodes
}
```